GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/scripts/Objects/ServerResponseModel.cs

```diff
@@ -0,0 +1,6 @@
+public class ServerResponseModel
+{
+    public bool success;
+    public string reason;
+    public string result;
+}
```

Comment:
Please add some getter methods. It might not very useful here but always a good coding style.
https://codeasy.net/lesson/properties
You can use Properties in C#.

Reply:
Yep